deparse.c

```diff
@@ -1845,6 +1853,60 @@ sqlite_deparse_range_tbl_ref(StringInfo buf, PlannerInfo *root, RelOptInfo *fore
 										 params_list);
 }
 
+/*
+ * Add a RETURNING clause, if needed, to an INSERT/UPDATE/DELETE.
+ */
+static void
+sqlite_deparseReturningList(StringInfo buf, PlannerInfo *root,
+							Index rtindex, Relation rel,
+							bool trig_after_row,
+							List *withCheckOptionList,
+							List *returningList,
+							List **retrieved_attrs)
+{
+	Bitmapset  *attrs_used = NULL;
+
+	elog(DEBUG3, "sqlite_fdw : %s", __func__);
+	if (trig_after_row)
+	{
+		/* whole-row reference acquires all non-system columns */
+		attrs_used =
+			bms_make_singleton(0 - FirstLowInvalidHeapAttributeNumber);
+	}
+
+	if (withCheckOptionList != NIL)
```

Comment:
Could you add test case to cover this condition?

Reply:
Is this about view after tested table https://www.postgresql.org/docs/17/sql-createview.html + https://www.postgresql.jp/document/16/html/fdw-callbacks.html#FDW-CALLBACKS-UPDATE ? Will this https://neon.tech/postgresql/postgresql-views/postgresql-views-with-check-option relevant TCs?